.github/actions/tests-linux-containerless/action.yaml

```diff
@@ -0,0 +1,72 @@
+name: CLI container-less on Linux
+
+inputs:
+  tag:
+    description: |
+      Tag release
+    required: false
+    type: string
+    default: latest
+  tier:
+    description: |
+      Test TIER name
+    required: false
+    type: string
+    default: TIER0
+
+runs:
+  using: "composite"
+  steps:
+  - uses: actions/checkout@v4
+  - uses: actions/setup-java@v4
+    with:
+      distribution: 'microsoft'
+      java-version: '21'
+  - run: java --version
+    shell: bash
+  - name: Set up Igor's kantra installer
+    shell: bash
+    run: |
+      git clone https://github.com/ibragins/konveyor-cli-deployment.git
+      cd konveyor-cli-deployment
+      pip install -r requirements.txt
+      echo '{"misc_downstream_path": "","temp_dir": "","extract_binary": "","get_images_output": "","bundle": "","no_brew": "","ssh_user": "","ssh_key": ""}' >> config.json
+      # Workaround ruleset home (?)
+      mkdir -p /home/runner/.config
+      ./install_cli.py --upstream true
+      cd ..
+      chmod +x /home/runner/.kantra/kantra
+      /home/runner/.kantra/kantra --help
+
+  - name: Configure Test Environment Variables
+    shell: bash
+    run: |
+      cp .env.example .env
+      mkdir ${{ github.workspace }}/report
+      
+      > .env
+      echo "KANTRA_CLI_PATH=/home/runner/.kantra/kantra" >> .env
+      echo "REPORT_OUTPUT_PATH=${{ github.workspace }}/report" >> .env
+      echo "PROJECT_PATH=${{ github.workspace }}" >> .env
```

Comment:
The .env file also need to configure the followin properties for analysis test on private repos:

- `GIT_USERNAME`  
- `GIT_PASSWORD`  

This can be configured in a follow-up PR as the test that you're running right now is not using these properties.

Reply:
Thanks for feedback!
- credentials - correct, should be setup in these workflows once we add an application that needs credentials (so likely in one of following PRs)
- failures - this PR adds github workflows, so those failures doesn't change any existing test. Based on our recent discussion, going to address those failures on a fresh simple analysis test case in https://github.com/konveyor-ecosystem/kantra-cli-tests/pull/48 (that most likely will modify test workflows to allow pass the given analysis test case)